src/components/screens/inviteController.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react";
+import Home from "./home";
+import FriendsModal from "./inviteFriendsModal";
+import { InviteApi } from "../../services/inviteApi";
+
+function InviteController() {
+  const [showFriendsModal, setShowFriendsModal] = useState(false);
+  const [friends, setFriends] = useState([]);
+  const [message, setMessage] = useState("");
+  const inviteApi = new InviteApi(
+    process.env.REACT_APP_APIURL,
+    process.env.REACT_APP_APICODE
+  );
+
+  useEffect(() => {
+    inviteApi.getCandidates().then((response) => {
+      setFriends(response);
+    });
+  }, []);
+
+  const toggleModal = () => {
+    setShowFriendsModal(!showFriendsModal);
+  };
+
+  const handleInvite = (selectedFriends) => {
+    inviteApi.sendInvites(selectedFriends).then((response) => {
+      setMessage(response.message);
+    });
+  };
+
+  let visibleScreen = null;
+  if (showFriendsModal) {
+    visibleScreen = (
+      <FriendsModal
+        exitClick={toggleModal}
+        friendsList={friends}
+        inviteFriends={handleInvite}
+      />
+    );
+  } else {
+    visibleScreen = <Home inviteClick={toggleModal} userMessage={message} />;
+  }
+  return <div>{visibleScreen}</div>;
```

Comment:
You could probably simplify this section slightly, like this:
```diff
-  let visibleScreen = null;
   if (showFriendsModal) {
-    visibleScreen = (
+    return (     
       <FriendsModal
         exitClick={toggleModal}
         friendsList={friends}
         inviteFriends={handleInvite}
       />
     );
-  } else {
-    visibleScreen = <Home inviteClick={toggleModal} userMessage={message} />;
-  }
-   return <div>{visibleScreen}</div>;
+   return <Home inviteClick={toggleModal} userMessage={message} />
```

Reply:
fixed